pkg/resources/external_function_acceptance_test.go

```diff
@@ -7,126 +7,158 @@ import (
 	"testing"
 
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func TestAcc_ExternalFunction(t *testing.T) {
+func TestAcc_ExternalFunction_basic(t *testing.T) {
```

Comment:
can we update args and test the behavior then? Also, let's test no args at all.

Reply:
added test with no_args. updating args would just be a force-new change, so I don't think that is something we need to test.

Reply:
We want it to be force new, so the test would document the expected behavior. I will close it either way.